### ISSUE TITLE
How to disable application re-render when connection state changes?

### ISSUE THREAD

**Issue by vikas26294** [rgommezz/react-native-offline#128]
I am checking connectivity at regular interval and when connectivity state changes, it re-renders the whole application. Is there any way to stop re-rendering part while keeping regular interval connectivity check?

```
Main = withNetworkConnectivity({
  withRedux: true,
  timeout: 3000,
  pingServerUrl: 'https://www.google.com/',
  withExtraHeadRequest: true,
  checkConnectionInterval: 3000,
  checkIntervalOfflineOnly: false,
  checkInBackground: false,
  httpMethod: 'HEAD',
})(Main);
```

**Comment by rgommezz** [comment]
Sorry, there was a regression. It should be fixed with `v3.14.1`

**Comment by incorelabs** [comment]
I think I'm facing a similar issue. I am using withNetworkConnectivity() without any parameters but as soon as the network status changes, it goes from online to offline, it shows the error component. How do I prevent this from automatically rendering. I only want it to check for internet connectivity manually.
